service-runner/src/main/resources/config/assets.yaml

```diff
@@ -14,6 +14,36 @@ assets:
       fee_fixed: 0
       fee_percent: 0
       max_amount: 1000000
+    sep6:
+      deposit:
+        exchange_enabled: true
```

Comment:
💡 I think we can always make the `*-exchange` endpoints available. Ideally, the fact that SEP-6 has a different set of endpoints for nonequivalent assets can be abstracted away.